### ISSUE TITLE
custom segmentation for 1 class and background

### ISSUE THREAD

**Issue by lattard** [divamgupta/image-segmentation-keras#131]
Hi @divamgupta many thanks for sharing this implementation for segmentation.

I have been experimenting with the UNET implementation in this repo and have some questions re my scenario, in which I have a single class + background and I am experimenting with a modified version of the unet_mini increasing the depth by another level

1. what should I set the number of classes to: 1 or 2?
2. can I also use the same activation as softmax at the end or should I change to sigmoid?
3. as for the loss should I still use categorical cross entropy or should I change to binary cross entropy instead?

Thanks

**Comment by divamgupta** [comment]
Two classes should work fine. Class 0 - background. Class 1 - foreground.
